Lombiq.Hosting.MediaTheme/Recipes/Lombiq.Hosting.MediaTheme.Samples.recipe.json

```diff
@@ -31,10 +31,6 @@
           "SourcePath": "_MediaTheme/Assets/example.png",
           "TargetPath": "_MediaTheme/Assets/example.png"
         },
-        {
-          "SourcePath": "_MediaTheme/Assets/favicon.ico",
-          "TargetPath": "_MediaTheme/Assets/favicon.ico"
-        },
```

Comment:
Using a separate image was good. Also to make sure that the browser attempts to load it.